Hdf5/Hdf5FileHeader.cpp

```diff
@@ -44,6 +44,11 @@
   #pragma comment(lib, "Ws2_32.lib")
 #endif
 
+// Apple build
+#ifdef __APPLE__
+  #include <unistd.h>
+#endif
+
```

Comment:
Unifying with the above guard.
```suggestion
// Unix-like systems (Linux and macOS)
#if defined(__linux__) || defined(__APPLE__)
  #include <unistd.h>
#endif

```